src/patternfly/components/Menu/menu.scss

```diff
@@ -76,6 +83,23 @@
   --pf-c-menu__item-action--m-favorite__icon--FontSize: var(--pf-global--icon--FontSize--sm);
   --pf-c-menu__item-action--m-favorite--m-favorited__icon--Color: var(--pf-global--palette--gold-400);
 
+  // Drilldown
+  --pf-c-menu--m-drilldown--Width: auto;
+  --pf-c-menu--m-drilldown--Height: auto;
+  --pf-c-menu--m-drilldown--Transition--transform: var(--pf-global--TransitionDuration);
+  --pf-c-menu--m-drilldown--Transition--height: var(--pf-global--TransitionDuration);
+
+  // Drilldown menu
+  --pf-c-menu--m-drilldown__menu--Transition--transform: var(--pf-global--TransitionDuration);
```

Comment:
The naming should include TransitionDuration and then assign it to transform below (check out Dropdown and Navigation)